src/cascadia/TerminalSettingsEditor/ColorSchemes.xaml

```diff
@@ -139,6 +141,23 @@ the MIT License. See LICENSE in the project root for license information. -->
                             <TextBlock Text="{x:Bind Name, Mode=OneWay}"
                                        Style="{StaticResource ColorHeaderStyle}"/>
                             <Button Background="{x:Bind Color, Converter={StaticResource ColorToBrushConverter}, Mode=OneWay}">
+
+
+                                <Button.Resources>
+                                    <!-- Resources to colorize hover/pressed states based on the color of the button -->
+                                    <ResourceDictionary>
+                                        <ResourceDictionary.ThemeDictionaries>
+                                            <ResourceDictionary x:Key="Light">
+                                                <SolidColorBrush x:Key="ButtonBackgroundPointerOver" Color="{x:Bind Color, Converter={StaticResource ColorLightenConverter}, Mode=OneWay}"/>
+                                            </ResourceDictionary>
+                                            <ResourceDictionary x:Key="Dark">
+                                                <SolidColorBrush x:Key="ButtonBackgroundPointerOver" Color="{x:Bind Color, Converter={StaticResource ColorLightenConverter}, Mode=OneWay}"/>

```

Comment:
I could be _completely_ wrong here because I don't have a design background, but I honestly don't know.

Should we create/use a `ColorDarkenConverter` for the `Dark` theme? I've noticed that regularly styled buttons respond differently on hover:
- darken in dark theme
- lighten in light theme

Though that may be accomplished by your alpha change?

Reply:
If curious, I did find [this post](https://graphicdesign.stackexchange.com/questions/75417/how-to-make-a-given-color-a-bit-darker-or-lighter#:~:text=Say%20in%20rgb%20you%20have%20132%2C145%2C167%20yo%20can,bit%20then%20subtract%202%20from%20each%20so%20160%2C173%2C205.) on how to lighten/darken a color. But again, this all might be accomplished by your alpha change.